CS342MemberViews/Sources/AshleyGriffin/Resources/AshleyGriffinBio.md

```diff
@@ -1 +1,7 @@
 Dr. Ashley Griffin is a medical informatics postdoctoral fellow in the Department of Health Policy at Stanford and the VA Palo Alto Health Care System. Her research focuses on the use of digital health technologies and patient-generated health data to facilitate shared decision-making and inform interoperability initiatives.
+
+**Hobbies**
+- Cross-stitch 🪡
+- Piano 🎹
+- Yoga 🧘‍♀️
+- Listening to Taylor Swift 🕛
```

Comment:
@griffinac you get extra points for this one 😊

Reply:
Agree that mentions of Taylor Swift deserve extra points 🤣